src/constraints/ramping-and-unit-commitment.jl

```diff
@@ -22,28 +22,31 @@ function add_ramping_constraints!(model, variables, constraints, graph, sets)
 
     ## Expressions used by the ramping and unit commitment constraints
     # - Expression to have the product of the profile and the capacity paramters
-    profile_times_capacity = [
-        @expression(
-            model,
-            profile_aggregation(
-                Statistics.mean,
-                graph[row.asset].rep_periods_profiles,
-                row.year,
-                row.year,
-                ("availability", row.rep_period),
-                row.time_block_start:row.time_block_end,
-                1.0,
-            ) * graph[row.asset].capacity
-        ) for row in eachrow(cons_with.indices) if is_active(graph, row.asset, row.year)
-    ]
+    profile_times_capacity = Dict(
+        key => begin
+            table_name = Symbol("ramping_$(key)_unit_commitment")
```

Comment:
Also, it reads better below if the keys are `:with_unit_commitment` and `:without_unit_commitment`

So, please change it either here or in the other PR you are working for the ramping. Thanks!

Reply:
I agree. I will change in the next PR, because I had to completely change the keys to the table names, since they are used in other tables as well